2context.vim

```diff
@@ -43,6 +45,34 @@ if !exists("escapecomments")
   let escapecomments=0
 endif
 
+if !exists("showlinetags")
+   let showlinetags=1
+endif
+let s:startcomment = []
+let s:stopcomment = []
+for s:comsym in split(&comments,',')
+  let s:comsym = split(s:comsym,':')
+  echo s:comsym
+  if len(s:comsym) < 2
+    let s:startcomment = add(s:startcomment,escape(s:comsym[0],'\[*.]^$'))
+  elseif ( s:comsym[0] =~ 's\|m\|^[^e]*$' || s:comsym[0] =~ '^$' )
+    let s:startcomment = add(s:startcomment,escape(s:comsym[1],'\[*.]^$'))
+  else
+    let s:stopcomment = add(s:stopcomment,escape(s:comsym[1],'\[*.]^$'))
+  endif
+endfor
+if len(s:startcomment) < 1 && exists("+commentstring")
+  let s:havecomment = split(&commentstring,"%s")
+  if len(s:havecomment) > 0
+    let s:startcomment = add(s:startcomment,s:havecomment[0])
+    if len(s:havecomment) > 1
+      let s:stopcomment = add(s:stopcomment,s:havecomment[1])
+    endif
+  endif
+endif
```

Comment:
To avoid a performance penalty, this should be wrapped in a `if(showlinetags)` block.

Reply:
Hi cause i first edited on edit and submitted to master and lateron switched to subbranch of dev mergingin my changes from master directly. I will change such that pullrequest only contains changes against dev.